source/app/service-providers/windows/dialog/ask-directory.ts

```diff
@@ -20,18 +20,22 @@ import { trans } from '@common/i18n-main'
  * Asks the user for directory path(s)
  *
  * @param   {BrowserWindow|null}  win  The window to attach to
- *
+ * @param title {string}               Title of the Window
+ * @param buttonLabel {string|null}    Label of the Button
  * @return  {Promise<string>[]}        Resolves with an array of paths
  */
-export default async function askDirectory (config: ConfigProvider, win: BrowserWindow|null): Promise<string[]> {
+export default async function askDirectory (config: ConfigProvider, win: BrowserWindow|null, title: string, buttonLabel: string|null): Promise<string[]> {
   let startDir = app.getPath('home')
 
   if (isDir(config.get('dialogPaths.askDirDialog'))) {
     startDir = config.get('dialogPaths.askDirDialog')
   }
-
+  if (buttonLabel == null) {
+    buttonLabel = trans('system.open_folder')
```

Comment:
Can't we use the Electron default here? I think that may be even better. It should even work out of the box if you make that buttonLabel optional and set it to undefined|string

Reply:
The issue with the electron-default is that it is "Open". This is not what we need for the export.

Using undefined is actually better, this way we can leave out the "default" translation-check.

Reply:
Ahhh I see. There is the equivalent of the save dialog, that works similarly but with save as default labels. We might be able to just add that to the `askDir` function

Reply:
Oh yeah, there is `SaveDialogOptions`. However, i would have to basically duplicate the whole askDir function to use that. I can do that, to remove the parameter, but i would have to duplicate quite a bit of code. It's up to you how you like me to handle this :D